.test-infra/jenkins/job_Metrics_Report.groovy

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import InfluxDBCredentialsHelper
+
+job('beam_Metrics_Report') {
```

Comment:
There is a Jenkins job that did the similar thing in the past: https://github.com/apache/beam/blob/master/.test-infra/jenkins/job_beam_PerformanceTests_Analysis.groovy. It's not being used at the moment. We should either bring it back to live (by modifying it with your changes) or remove it completely.

Reply:
https://github.com/apache/beam/pull/12728